.gitignore

```diff
@@ -1,6 +1,7 @@
 # IDE
 .idea
 .vscode
+.history
```

Comment:
Why did you add this?

Reply:
I had an unpleasant case, when I had to remove half of the last changes made after last commit. After that, I add HISTORY extension in VSC - just for myself.  I don't push it on the git hub because there is hundreds of files.